clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/ServerStoreOpMessage.java

```diff
@@ -23,12 +23,12 @@
 public abstract class ServerStoreOpMessage extends EhcacheEntityMessage implements ConcurrentEntityMessage {
   public enum ServerStoreOp {
 
-    GET((byte) 10),
-    GET_AND_APPEND((byte) 11),
-    APPEND((byte) 12),
-    REPLACE((byte) 13),
-    CLIENT_INVALIDATION_ACK((byte) 14),
-    CLEAR((byte) 15),
+    GET((byte) 11),
+    GET_AND_APPEND((byte) 12),
+    APPEND((byte) 13),
+    REPLACE((byte) 14),
+    CLIENT_INVALIDATION_ACK((byte) 15),
+    CLEAR((byte) 16),
```

Comment:
With the introduction of the state repo operations, one more top level message got introduced, in addition to life-cycle messages and server-store messages. So I thought I'll categorize them into 3 buckets:

01<value<= 10 : life cycle messages
11<value<=20: server store messages
21<value<=30: state repo messages

This was based on the assumption that the sub-level message won't go beyond 10.
So basically I marked the upper bounds: 10, 20 and 30 and used those upper bound values as the enum values for the top-level message types defined in `EhcacheEntityMessage.Type`
And then there was a clash: `EhcacheEntityMessage.Type.LIFECYCLE_OP` and `ServerStoreOpMessage.ServerStoreOp.GET` had the same value: 10. So to avoid that clash I advanced all the `ServerStoreOpMessage.ServerStoreOp` types starting from 11.

I could have made the upper bound of life cycle message type as 9. But then the sequence of upper bounds looked weird: 9, 20 and 30. That's why I didn't do that. But I could revert to that if that's fine.

Or there is something even better that I could do. Just change the value of `ServerStoreOpMessage.ServerStoreOp.GET` to 16 from 10 instead of changing the value of all enums. So I think that's the best thing to do here.